Data/Primitive/Ptr.hs

```diff
@@ -25,10 +26,15 @@ module Data.Primitive.Ptr (
 
   -- * Block operations
   copyPtr, movePtr, setPtr
+
+#if __GLASGOW_HASKELL__ >= 708
+  , copyPtrToMutablePrimArray
+#endif
 ) where
 
 import Control.Monad.Primitive
 import Data.Primitive.Types
+import Data.Primitive.PrimArray (MutablePrimArray(..))
```

Comment:
This should be guarded with `#if __GLASGOW_HASKELL__ >= 708`, right?

Reply:
I'll fix this. I've never been totally clear on the policy around warnings though. Is it something like "avoid any warnings on any version of GHC"?

Reply:
> Is it something like "avoid any warnings on any version of GHC"?

Ideally that's be the case, yes. I say "ideally" since `primitive` doesn't currently meet this standard—it emits several warnings on older versions of GHC. If we really wanted to uphold this, we should enable `-Werror` and weed out the existing warnings.

That being said, this particular warning has a very simple fix, so I thought it prudent to catch it while I still could.